src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsnonassociationsapi/dto/SyncRequest.kt

```diff
@@ -0,0 +1,116 @@
+package uk.gov.justice.digital.hmpps.hmppsnonassociationsapi.dto
+
+import io.swagger.v3.oas.annotations.media.Schema
+import uk.gov.justice.digital.hmpps.hmppsnonassociationsapi.SYSTEM_USERNAME
+import uk.gov.justice.digital.hmpps.hmppsnonassociationsapi.jpa.NonAssociation
+import java.time.LocalDate
+
+@Schema(description = "Create Sync Request")
+data class CreateSyncRequest(
+  @Schema(description = "Prisoner number to not associate", required = true, example = "A1234BC")
+  val firstPrisonerNumber: String,
+  @Schema(
+    description = "Reason why this prisoner should be kept apart from the other",
+    required = true,
+    example = "VICTIM",
+  )
+  val firstPrisonerReason: NonAssociationReason,
+  @Schema(description = "Prisoner number to not associate", required = true, example = "D5678EF")
+  val secondPrisonerNumber: String,
+  @Schema(
+    description = "Reason why this prisoner should be kept apart from the other",
+    required = true,
+    example = "PERPETRATOR",
+  )
+  val secondPrisonerReason: NonAssociationReason,
+
+  @Schema(description = "Type of restriction, e.g. don't locate in the same cell", required = true, example = "CELL")
+  val restrictionType: NonAssociationRestrictionType,
+
+  @Schema(
+    description = "Explanation of why prisoners are non-associated",
+    required = false,
+    example = "John and Luke always end up fighting",
+  )
+  val comment: String? = null,
```

Comment:
why is the comment optional? does NOMIS make a distinction between empty and null here?

Reply:
Its a nullable field in NOMIS